### PR TITLE
Changed some CSS for responsiveness and added link to the button

### PR DESCRIPTION
**Pull request by sandhana14**
# Description

Changed some CSS  style for responsiveness of the error page  and added a link to the button

Fixes # (issue)
#23 

# How to test

First, import the error page component in the app.js and add a route for it and then test by running npm run dev

# Checklist

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by TITHI1244 — APPROVED** [review]
To me, the page does not look quite same for very big screens such as over 1800 width. Maybe you can consider using media query :) Good job @sandhana14

**Comment by sandhana14** [comment]
> To me, the page does not look quite same for very big screens such as over 1800 width. Maybe you can consider using media query :) Good job @sandhana14

**Comment by TITHI1244** [comment]
Seems exactly perfect for small and big both screens :)

**Review by rajanibusani — COMMENTED** [review]
A very good job @sandhana14 , Your error page is responsive in all desktop screen sizes...👏

**Review by rajanibusani — APPROVED** [review]

**Review by rajanibusani — COMMENTED** [review]
Hi @ccshah298 Good job Charmi..👏

**Comment by ccshah298** [comment]
good job @sandhana14

**Review by bytewiz — APPROVED** [review]

File: src/client/containers/404Page/404Page.container.js [hunk 1]
```diff
@@ -21,7 +22,9 @@ function Error404Page() {
         </div>
 
         <div>
-          <Buttons label="Go Home" size="big" isMono={true} />
+          <Link to="/">
+            <Buttons label="Go Home" size="big" isMono={true} />
+          </Link>
```
Comment (bytewiz) [inline src/client/containers/404Page/404Page.container.js:27]: This is not the correct way to write this. think of it as it ends up being HTML and you should not wrap an <a> tag around a <button> tag.

https://stackoverflow.com/questions/6393827/can-i-nest-a-button-element-inside-an-a-using-html5

If you want to navigate to a page using a button this is the correct way to do so :).

https://reactrouter.com/web/api/Hooks/usehistory

**Review by bytewiz — CHANGES_REQUESTED** [review]

**Review by bytewiz — APPROVED** [review]
Perfect 👌

**Pull request merged**